### PR TITLE
PEG extension for case insensitive literals.

### PR DESCRIPTION
**Pull request by veelo**
Hi Philippe,

This implements the suggestion made by Bit-Doctor in #119. It supports unicode.

Hope you like it,
Bastiaan.

### CODE REVIEW

File: pegged/docs/wiki [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit f111de4928c41cc22d2d093e3362f00bf7f80733
+Subproject commit 69188d442c1e7e4ad3976e5fc3988792723fc097
```
Comment (veelo) [inline pegged/docs/wiki]: I didn't know how github wikis work yet, so I mistakenly had a reference here to a branch in a local clone of your wiki. I don't think that would work, so this is a reference to the latest commit in origin of your wiki instead; I suppose that is correct. Somehow I couldn't revert to what it was before, but that reference was rather old anyway.

I'll amend the PEG-extensions page accordingly online if you merge this.

**Comment by PhilippeSigaud** [comment]
Oh, I did not even know about `icmp`! Good find, that's something many people asked for.
Thanks a lot, I'll merge that.

File: pegged/docs/wiki (hunk 1 above)
Comment (PhilippeSigaud) [inline pegged/docs/wiki]: What part of the wiki did you change? I had a look in the 'Extended Pegged Syntax' section, but did not find the "abc"i syntax you introduced.

Comment (veelo) [inline pegged/docs/wiki]: First I changed it offline in a local checkout of the wiki source as a git subproject. Then I realised that the wiki isn't meant to be used like that.

I just edited the wiki online: https://github.com/PhilippeSigaud/Pegged/wiki/Extended-PEG-Syntax#case-insensitive-literals

**Comment by veelo** [comment]
Yeah at first I had it implemented using toLower, and found icmp just before pushing to github.

Thanks for the merger.

File: pegged/docs/wiki (hunk 1 above)
Comment (PhilippeSigaud) [inline pegged/docs/wiki]: I see it, that works. Thanks.

**Pull request merged**